e2e_tests/tests/cluster/test_task.py

```diff
@@ -72,3 +73,34 @@ def test_task_get_config() -> None:
         """
         )
         assert sorted(expected.items()) == sorted(result.items())
+
+@pytest.mark.e2e_cpu
+def test_task_fork() -> None:
+    sess = api_utils.determined_test_session(admin=True)
+    with open("e2e_tests/test_config.yaml", "r") as config_file:
```

Comment:
this is dicey because it'd break whenever someone runs the e2e tests from any other directory other than the project root. use helpers like [conf.fixtures_path](https://github.com/determined-ai/determined/blob/main/e2e_tests/tests/config.py#L48) or construct a custom path from [PROJECT_ROOT_PATH](https://github.com/determined-ai/determined/blob/main/e2e_tests/tests/config.py#L36)